bench/lmdb_bench.ml

```diff
@@ -0,0 +1,237 @@
+(** Benchmarks *)
+
+open Common
+open Rresult
+
+let src = Logs.Src.create "db_bench"
+
+module Log = (val Logs.src_log src : Logs.LOG)
+
+let seed = 1
+
+let () = Random.init seed
+
+let key_size = 32
+
+let hash_size = 30
+
+let value_size = 13
+
+let nb_entries = 10_000_000
+
+let log_size = 500_000
+
+let ( // ) = Filename.concat
+
+module Context = Make_context (struct
+  let key_size = key_size
+
+  let hash_size = hash_size
+
+  let value_size = value_size
+end)
+
+let entry_size = key_size + value_size
+
+let random = Array.make nb_entries ("", "")
+
+let populate () =
+  let rec loop i =
+    if i = nb_entries then ()
+    else
+      let k = Context.Key.v () in
+      let v = Context.Value.v () in
+      let () = random.(i) <- (k, v) in
+      loop (i + 1)
+  in
+  loop 0
+
+let print_results db f nb_entries =
+  let _, time = with_timer f in
+  let micros = time *. 1_000_000. in
+  let sec_op = micros /. float_of_int nb_entries in
+  let mb = float_of_int (entry_size * nb_entries / 1_000_000) /. time in
+  let ops_sec = float_of_int nb_entries /. time in
+  Log.app (fun l ->
+      l "%s: %f micros/op; \t %f op/s; \t %f MB/s; \t total time = %fs." db
+        sec_op ops_sec mb time)
+
+module Lmdb = struct
```

Comment:
There are a few things in the `lmdb_bench.ml` that are common to the `index_bench.ml`: can you keep only the `lmdb` module here, and then call it from the `index_bench`?  Then you will also have only one executable. 

In particular the data set I think should be shared between the different benchmarks.